geminimd/commands.py

```
@@ -0,0 +1,401 @@
+
+
+def list_command(options: ListOptions, env: CommandEnv) -> ListResult:
+    drive = env.drive or DriveClient(env.ui)
+    env.drive = drive
+    chats = drive.list_chats(options.folder_name, options.folder_id)
+    chats = filter_chats(chats, options.name_filter, options.since, options.until)
+    folder_id = drive.resolve_folder_id(options.folder_name, options.folder_id)
+    return ListResult(folder_name=options.folder_name or DEFAULT_FOLDER_NAME, folder_id=folder_id, files=chats)
+
+
+def sync_command(options: SyncOptions, env: CommandEnv) -> SyncResult:
```

Ruff (0.15.9)

[warning] Too many branches (22 > 12)

(PLR0912)